.github/actions/changelog/entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+
+# simple changelog: get all commits beetween 2 tags and put them into file
+
+echo "Commits: " > changelog.md
+#find previos tag
+start_tag=$(git for-each-ref refs/tags/ --count=2 --sort=-version:refname --format='%(refname:short)' | awk 'NR==2')
```

Comment:
[q] can the `git describe --abbrev=0 --tags`give the same output?

Reply:
*_*  `git describe --abbrev=0 --tags --match 'v*'` looks like the same, thanks 👍🏼